### PR TITLE
Senior Software Engineer, Databases

### PR DESCRIPTION
**Pull request by ebuchman**
Not sure if we want to include stuff about Amino and ABCI in here, or if that should be another posting.

### CODE REVIEW

File: md/careers/technical-senior-software-engineer-db.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* A strong grasp of computer science fundamentals. 
+* Deep understanding of the theory and implementation of database trees on real file systems.
+* Experience working in an agile development environment.
+* The ability to take ownership and see initiatives through.
+* Worked with and mentored junior engineers.
+* Exceptional communication skills.
+* Experience working with distributed teams.
+
+## What your primary responsibilities will be:
+* Improve the IAVL codebase by committing DRY, well tested code.
```
Comment (zramsay) [inline md/careers/technical-senior-software-engineer-db.md]: dry?

Comment (nylira) [reply md/careers/technical-senior-software-engineer-db.md]: don't repeat yourself - well-refactored, non-redundant code

File: md/careers/technical-senior-software-engineer-db.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Work with the Cosmos-SDK team to keep the IAVL tree and its proofs well integrated.
+* Optimize our implementation of the IAVL tree and its proofs.
+* Research, design, and implement improvements to the IAVL tree, the proofs, and related protocols. 
+* Help formalize the specification of the IAVL tree and proofs and ensure the implementation adheres to the spec.
+* Develop simulations for testing the behaviour of the IAVL tree under various loads and read/write strategies.
+* Develop alternatives to the IAVL tree like the Ethereum Patricia Trie that adhere to the same interfaces.
+* Mentor junior engineers and support the growth and health of the team.
+* Review pull requests and provide valuable feedback to members of the team and open source contributors.\
```
Comment (jolesbi) [inline md/careers/technical-senior-software-engineer-db.md]: remove this rogue slash

File: md/careers/technical-senior-software-engineer-db.md (hunk 1 above)
Comment (jolesbi) [inline md/careers/technical-senior-software-engineer-db.md]: can we add links to the repo's we're referencing?

**Pull request merged**